src/ray/core_worker/core_worker.cc

```diff
@@ -299,7 +299,7 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
   core_worker_server_ = std::unique_ptr<rpc::GrpcServer>(
       new rpc::GrpcServer(WorkerTypeString(options_.worker_type), assigned_port));
   core_worker_server_->RegisterService(grpc_service_);
-  core_worker_server_->Run();
+  core_worker_server_->Run(/*num_retries=*/1);
```

Comment:
Can we just set this by default instead of having to pass it in every time? I think we want a relatively long timeout for all of the servers

Reply:
We can always parametrize it later if need be

Reply:
Okay!